README.md

```diff
@@ -170,8 +170,8 @@ This plugin uses ruby-kafka producer for writing data. This plugin works with re
 
       default_topic         (string) :default => nil
       default_partition_key (string) :default => nil
-      output_data_type      (json|ltsv|msgpack|attr:<record name>|<formatter name>) :default => json
-      output_include_tag    (bool) :default => false
+      output_data_type      (json|ltsv|msgpack|single value|attr:<record name>|<formatter name>) :default => {"message": [json]}
```

Comment:
This change is incorrect.
default is `json` and kafka plugin doesn't put `{"message": [json]}` format.